packages/esm-patient-attachments-app/src/camera-media-uploader/file-review.component.tsx

```diff
@@ -12,8 +12,7 @@ export interface FileReviewContainerProps {
 }
 
 const FileReviewContainer: React.FC<FileReviewContainerProps> = ({ onCompletion }) => {
-  const { filesToUpload, clearData, setFilesToUpload, closeModal, collectDescription } =
-    useContext(CameraMediaUploaderContext);
+  const { filesToUpload, clearData, setFilesToUpload, closeModal, collectDescription } = useContext(CameraMediaUploaderContext);
```

Comment:
Nothing seems to have changed here, check the [main branch](https://github.com/openmrs/openmrs-esm-patient-chart/blob/1ce78c8a78e33f6668fd901d61f6bfb23966a89c/packages/esm-patient-attachments-app/src/camera-media-uploader/file-review.component.tsx#L15) as well

Reply:
You're right! I didn't notice in the previous commit that I caused the code line to become [two](https://github.com/openmrs/openmrs-esm-patient-chart/commit/1ce78c8a78e33f6668fd901d61f6bfb23966a89c#diff-bc0178b3955bb6dfa35eec6a275c7da0a5ea3e1c5f247da211a3a6ae209637ceL15) and I think it's more better when is a single line that is why I suggested to revert it back to one code line. However, if it's more alright I can revert it back.